rs/tests/src/ckbtc/btc_config.rs

```diff
@@ -23,7 +23,7 @@ cp /config/bitcoin.conf /tmp/bitcoin.conf
 docker run  --name=bitcoind-node -d \
   --net=host \
   -v /tmp:/bitcoin/.bitcoin \
-  registry.gitlab.com/dfinity-lab/open/public-docker-registry/kylemanna/bitcoind -rpcbind=[::]:8332 -rpcallowip=::/0
+  ghcr.io/dfinity/bitcoind@sha256:17c7dd21690f3be34630db7389d2f0bff14649e27a964afef03806a6d631e0f1 -rpcbind=[::]:8332 -rpcallowip=::/0
```

Comment:
@basvandijk, here I went with GHCR because otherwise we'll end up with issues with docker.

Reply:
Yes, all the once used in [uvm_config_image(...)](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/dfinity/ic%24+uvm_config_image%28&patternType=keyword&sm=0)

Reply:
Ok, but this happens in our CI/devenv context and not inside UVM on runtime like in the example above.

Reply:
Oh yes indeed. Actually that script above should use the [btc_uvm_config_image](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/tests/BUILD.bazel?L154) image and not download that image each time at runtime.

Reply:
It would be good to go over [all `docker run` invocations under rs/tests](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/dfinity/ic%24+file:rs/tests+%22docker+run%22&patternType=keyword&sm=0) and check that they don't pull images at runtime.

CC @nmattia who was also working on something related (making sure all the `docker run` invocations use the image from the preceding `docker load`).

Reply:
I briefly went over `docker run` invocations and it seems images are added to config image and loaded on the UVM. Only `rs/tests/src/custom_domains_integration/setup.rs` seems to pull `coredns`, `cloudlfare-api`, `pebble` and `openssl` images. Test `custom_domains_integration_test` is tagged with *manual* tag so I think we're good.

Reply:
@marko-k0 understanding question: Is that not a different image than the one that was previously used `registry.gitlab.com/dfinity-lab/open/public-docker-registry/kylemanna/bitcoind`? Looking at the hashes shown [here](https://hub.docker.com/r/kylemanna/bitcoind/tags), I don't see anything starting with `17c7dd`.

Reply:
> Only rs/tests/src/custom_domains_integration/setup.rs seems to pull coredns, cloudlfare-api, pebble and openssl images. 

Even they [load](https://sourcegraph.com/github.com/dfinity/ic@0b3c422965226ff0f4c2c4fd1e82811ac92cfcb8/-/blob/rs/tests/src/custom_domains_integration/activate.sh) the images from [the config image](https://sourcegraph.com/github.com/dfinity/ic@0b3c422965226ff0f4c2c4fd1e82811ac92cfcb8/-/blob/rs/tests/BUILD.bazel?L297-L313).

Reply:
@gregorydemay, I've checked whether the image is available in dockerhub (docker pull) and seeing it is, I only made the change to pull it from dockerhub in https://github.com/dfinity/ic/pull/1261. It seems that that image doesn't have any tag anymore on dockerhub. But it is the same image as we've been using for a while. This images are immutable ~ that's why we use sha256 notation instead of tag. Feel free to update the image in a separate PR and let me know if my wording here made things more confusing then clear.

Reply:
Thanks @marko-k0 for your explanation!